TuringAdvisoryGroup/tryrolljs
# Fix contract typings

## What's done

Currently, `dist` of the `contract-bindings` is missing some types because of no root folder. I have

- created a source (`src`) folder for the code
- updated rollup config to correctly generate the typings in `dist`.

## How to test

1. Run `yarn build`
2. Ensure that `contract-bindings/dist` has no missing type imports.

## Tasks


## Demo (screenshots, recordings, etc.)
